rewrite-test/src/main/kotlin/org/openrewrite/java/cleanup/UnnecessaryParenthesesTest.kt

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2020 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.cleanup
+
+import org.junit.jupiter.api.Disabled
+import org.junit.jupiter.api.Test
+import org.openrewrite.Recipe
+import org.openrewrite.RecipeTest
+import org.openrewrite.java.JavaParser
+import org.openrewrite.java.style.IntelliJ
+import org.openrewrite.java.style.UnnecessaryParenthesesStyle
+import org.openrewrite.style.NamedStyles
+
+interface UnnecessaryParenthesesTest : RecipeTest {
+    override val recipe: Recipe?
+        get() = UnnecessaryParentheses()
```

Comment:
Can equivalently be written in the simpler form `override val recipe = UnnecessaryParentheses()`

Reply:
`Property initializers are not allowed in interfaces` as the tests are all as interfaces extending `RecipeTest`:

```
interface RecipeTest {
    val recipe: Recipe?
        get() = null
...
```

so the hoop to jump through is to convert the property to a getter and have it return the desired recipe
![Screen Shot 2021-01-29 at 12 26 41 PM](https://user-images.githubusercontent.com/5619476/106313117-4b38dc00-622d-11eb-9d9c-72f4e9bf9083.png) 👍